posthog/temporal/data_imports/pipelines/pipeline_sync.py

```diff
@@ -79,8 +82,8 @@ def validate_schema_and_update_table_sync(
         table_format: The format of the table
         table_schema_dict: The schema of the table
     """
-
-    logger = bind_temporal_worker_logger_sync(team_id=team_id)
+    bind_contextvars(team_id=team_id)
+    logger = LOGGER.bind()
```

Comment:
Nit: it would be nice to be able to do this in a single function call - `logger = LOGGER.bind(team_id=team_id)`

Reply:
It's not semantically equivalent: `bind_contextvars` binds `team_id` to the local context, where as `LOGGER.bind` only binds it to the logger itself.

In the first case, any logger in the same local context (even across threads and asyncio tasks) will have `team_id` bound.

In the second case, the bind only applies to the one logger object. 

Anyways, all that being said, I checked and the function we are replacing here `bind_temporal_worker_logger_sync` did something closer to `LOGGER.bind`. So, I'll swap this to use `LOGGER.bind` just to stick closer to the old behavior.